pkg/serverlib/capability.go

```diff
@@ -119,7 +119,7 @@ func InstallCapability(client client.Client, mapper discoverymapper.DiscoveryMap
 		}
 	case types.TypeTrait:
 		var td v1alpha2.TraitDefinition
-		traitdata, err := ioutil.ReadFile(filepath.Clean(filepath.Join(repoDir, tp.CrdName+".yaml")))
+		traitdata, err := ioutil.ReadFile(filepath.Clean(filepath.Join(repoDir, tp.Name+".yaml")))
```

Comment:
what about the workload in L94

Reply:
I keep all code related to workload unchanged. So workloadDef file is still stored as CRD name as well as read.